src/Spectre.libGaClassifier/AllLabelTypesIncludedCondition.h

```diff
@@ -0,0 +1,55 @@
+/*
+* AllLabelTypesIncludedCondition.h
+* Checks, if Individual has at least specific amount of true values.
+*
+Copyright 2018 Spectre Team
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#pragma once
+#include "Spectre.libGenetic/BaseIndividualFeasibilityCondition.h"
+#include "Spectre.libClassifier/NotABinaryLabelException.h"
+#include <packages/Microsoft.Gsl.0.1.2.1/build/native/include/span.h>
```

Comment:
This should be included just like `<span.h>`.

Reply:
done